assets/css/index.css

```diff
@@ -41,152 +51,191 @@ main{
-    border-radius: 5px;
-    display: inline-block;
-    transition: background .3s ease;
-    margin-right: 5px;
+.nav-item-link:hover {
+  border: 1px solid black;
+  border-radius: 40px;
+  background-color: black;
 }
-.test{
-    background-color: #F9A826;
+
+.mobile-nav {
+  display: none;
+  cursor: pointer;
 }
-.practice{
-    background-color: #0198E1;
+.cancel {
+    display: none;
+    cursor: pointer;
 }
-.test:hover{
-    background-color: #f9a826d5;
+
+#hero,
+#features {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  flex-direction: row;
+  background-color: white;
+  border-radius: 30px;
+  margin: 0 30px 20px 30px;
+  padding: 10px;
+  height: max-content;
+}
+.left-section,
+.right-section {
+  padding: 5vw;
+  width: 35vw;
+  margin: auto;
+}
+.left-section h1 {
+  font-size: xxx-large;
+  font-weight: bolder;
+}
+.subtitle {
+  padding-block: 10px;
+  font-weight: 400;
+  font-size: large;
+}
+.right-section img {
+  width: 100%;
+  height: 100%;
+  background-position: center;
+  background-size: contain;
+}
+.heroBtn {
+  padding: 10px;
+  margin: 20px 0 10px 0;
+  color: white;
+  text-decoration: none;
+  border-radius: 5px;
+  display: inline-block;
+  transition: background 0.3s ease;
+  margin-right: 5px;
+}
+.test {
+  background-color: #f9a826;
+}
+.practice {
+  background-color: #0198e1;
+}
+.test:hover {
+  background-color: #f9a826d5;
 }
-.practice:hover{
-    background-color: #0196e1c7;
+.practice:hover {
+  background-color: #0196e1c7;
 }
 
-#features{
-    justify-content: space-around;
-    text-align: center;
-    padding-block: 20px;
+#features {
+  justify-content: space-around;
+  text-align: center;
+  padding-block: 20px;
 }
-.feature{
-    width: 25vw;
+.feature {
+  width: 25vw;
 }
-.feature img{
-    height: 96px;
-    width: 96px;
+.feature img {
+  height: 96px;
+  width: 96px;
 }
-.feature p{
-    padding: 5px;
+.feature p {
+  padding: 5px;
 }
 
 footer {
-    padding: 20px 0px;
-    font-weight: 500;
-    bottom: 0;
-    width: 100%;
-    font-size: 18px;
-    color: white;
-    display: flex;
-    justify-content: center;
-    background: rgb(2,0,36);
-background: linear-gradient(279deg, rgba(2,0,36,1) 0%, rgba(5,5,85,1) 52%, rgba(0,212,255,1) 100%);
-  }
+  padding: 20px 0px;
+  font-weight: 500;
+  bottom: 0;
+  width: 100%;
+  font-size: 18px;
+  color: white;
+  display: flex;
+  justify-content: center;
+  background: rgb(2, 0, 36);
+  background: linear-gradient(
+    279deg,
+    rgba(2, 0, 36, 1) 0%,
+    rgba(5, 5, 85, 1) 52%,
+    rgba(0, 212, 255, 1) 100%
+  );
+}
 
 /* ============ Media Queries ============ */
 
-@media screen and (max-width:670px){
-    .nav-item{
-        display: inline-flex;
-    }
-    .navigation{
-        margin: auto;
-    }
-    .brand-name{
-        margin-bottom: 10px;
-    }
-    .brand{
-        margin: auto;
-    }
-    footer{
-        font-size: 12px;
-    }
+@media screen and (max-width: 513px) {
```

Comment:
increase max-width to `525 px` as it is breaking the navbar between 513px and 525px.

Reply:
Updated